example/centos/centos-9-aarch64-image-installer.yaml

```diff
@@ -0,0 +1,572 @@
+                to: mount://device/
+          devices:
+            device:
+              type: org.osbuild.loopback
+              options:
+                filename: LiveOS/rootfs.img
+          mounts:
+            - name: device
+              type: org.osbuild.ext4
+              source: device
+              target: /
+    - name: efiboot-tree
+      build: name:build
+      stages:
+        - type: org.osbuild.grub2.iso
+          options:
+            product:
+              name: CentOS Stream
+              version: 9-stream
+            kernel:
+              dir: /images/pxeboot
+              opts:
+                - inst.stage2=hd:LABEL=CentOS-Stream-9-BaseOS-aarch64
+                - inst.ks=hd:LABEL=CentOS-Stream-9-BaseOS-aarch64:/osbuild.ks
+            isolabel: CentOS-Stream-9-BaseOS-aarch64
+            architectures:
+              - AA64
+            vendor: centos
+    - name: os
+      build: name:build
+      stages:
+        - otk.external.osbuild-make-depsolve-dnf4-rpm-stage:
+            packageset: ${packages.os}
+            gpgkeys:
+              otk.include: "common/gpgkeys.yaml"
+        - type: org.osbuild.fix-bls
+          options: {}
+        - type: org.osbuild.locale
+          options:
+            language: C.UTF-8
+        - type: org.osbuild.timezone
+          options:
+            zone: America/New_York
+        - type: org.osbuild.sysconfig
+          options:
+            kernel:
+              update_default: true
+              default_kernel: kernel
+            network:
+              networking: true
+              no_zero_conf: true
+        - type: org.osbuild.selinux
+          options:
+            file_contexts: etc/selinux/targeted/contexts/files/file_contexts
+    - name: bootiso-tree
+      build: name:build
+      stages:
+        - type: org.osbuild.mkdir
+          options:
+            paths:
+              - path: /images
+              - path: /images/pxeboot
+        - type: org.osbuild.copy
+          inputs:
+            tree:
+              type: org.osbuild.tree
+              origin: org.osbuild.pipeline
+              references:
+                - name:anaconda-tree
+          options:
+            paths:
+              - from: input://tree/boot/vmlinuz-8-2.fk1.aarch64
+                to: tree:///images/pxeboot/vmlinuz
+              - from: input://tree/boot/initramfs-8-2.fk1.aarch64.img
+                to: tree:///images/pxeboot/initrd.img
+        - type: org.osbuild.squashfs
+          inputs:
+            tree:
+              type: org.osbuild.tree
+              origin: org.osbuild.pipeline
+              references:
+                - name:rootfs-image
+          options:
+            filename: images/install.img
+            compression:
+              method: xz
+              options:
+                bcj: arm
+        - type: org.osbuild.truncate
+          options:
+            filename: images/efiboot.img
+            size: '20971520'
```

Comment:
Similar question about size as string vs number here as above and a bit of a meta question if we need an external or tweak the existing partition external for this block here, it seems it's doing a bunch of size adjustments that seem more suitable to an external(?)

Reply:
Moved into `${size_efiboot}` with a comment mentioning that it might need an external.